hypervisor/base/types.go

```diff
@@ -22,7 +22,7 @@ type Hypervisor interface {
 
 	Modify(vmName string, cpus int, mem int) error
 
-	HealthCheck(vmName, property string) (*string, error)
+	HealthCheck(vmName string) (*string, error)
```

Comment:
Why return *string? the string you assert on in the ip.go ("UP") is VirtualBox specific.

better to make it general so it is either pass (return on error) or return an error

Reply:
agreed making this return boolean instead of string